src/components/orders-table/listOrderDetails.component.tsx

```diff
@@ -0,0 +1,123 @@
+import React from "react";
+import styles from "./listOrderDetails.scss";
+
+import { useTranslation } from "react-i18next";
+import { showModal } from "@openmrs/esm-framework";
+import { Button, Tile } from "@carbon/react";
+import { OrderDetail } from "./orderDetail.component";
+import { ListOrdersDetailsProps } from "../../types";
+import { launchOverlay } from "../overlay/store";
+import ResultForm from "../../results/result-form.component";
+
+// can render orders of a patient
+const ListOrderDetails: React.FC<ListOrdersDetailsProps> = (props) => {
```

Comment:
Since we already have the types for the props, we should just destructure it e.g `({ actions })`